3_javascript/javascript3/week2/currencies.js

```diff
@@ -0,0 +1,163 @@
+const currencies = [
```

Comment:
You should get this currencies array from the API, since you already have that information in the response.